app/Api/Puzzles.ts

```diff
@@ -73,6 +78,20 @@ export class Puzzles {
       statistics.totalSolveTime / statistics.numGamesPlayed
     );
 
+    for (const newHintStrategies of numHintsUsedPerStrategy) {
```

Comment:
nit: single lint comment about this summing up what it does would be nice

Reply:
I added a comment there